builtin/game/item.lua

```diff
@@ -665,6 +665,10 @@ function core.itemstring_with_color(item, colorstring)
 	return stack:to_string()
 end
 
+function core.get_short_description(item)
+	return ItemStack(item):get_short_description()
+end
```

Comment:
Why do we need a function for this?

Reply:
I can just move this to the documentation instead. I wanted to make it harder for mods to check the def than to do it correctly

Reply:
Yes, please do.